repos.txt

```diff
@@ -5,13 +5,15 @@ java-analytics-admin
 java-analytics-data
 java-api-gateway
 java-apigee-connect
+java-apigee-registry
```

Comment:
It's GAPIC auto-generated library. It copies code from googleapis-gen (configuration: https://github.com/googleapis/java-apigee-registry/blob/main/.github/.OwlBot.yaml)